types/yanzi/comfort/uplink.js

```diff
@@ -0,0 +1,38 @@
+function consume(event) {
+  const { resourceType } = event.data.list[0];
```

Comment:
emit a sample for each of the values

Reply:
I never saw more than one value in the "list". But i might as well loop over it